tslint.report.active.json

```diff
@@ -923,6 +916,7 @@
   },
   "switch-final-break": {
     "documentation": "https://palantir.github.io/tslint/rules/switch-final-break",
+    "hasFix": true,
```

Comment:
Yes, but I would be very careful with running this script. especially when there are a lot of errors.
At least you need to run tslint again afterwards, since there is no guarantee that the fix for one rule isn't causing an error for another one.